ui/src/app/shared/header/header.component.ts

```diff
@@ -246,39 +233,3 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
   }
 }
 
```

Comment:
I could be missing something, but it looked like there was customized pagination functionality to handle the fact that pagination wasn't working correctly on the backend for Cromwell. Since I'm hoping to fix this with this PR, I removed it.

Reply:
That's not the purpose of this code. I will try to explain:
- QueryJobs is a paginated API.
- It is very common (at least for dsub), that when you come to the list page, your query will match way more results than you'd like to load (thousands, 10s of thousands), so we only load an initial hundred or so.
- (once you paginate, we will lazily load more pages as you near the end of what you have on the client)
- So a common state is that I've loaded ~100 / ~1k jobs that match my filter onto the client (in the client code, this is the "exhaustive" property).
- While we're in this state, we don't want to show "showing page 1/2", as this implies that your filter matches 100 jobs only.
- Instead, we have this custom code to override the default paginator text, and instead show "of many", since we don't have an accurate count of the total number of jobs.

Note: this behavior could actually be fixed in Cromwell's case by adding a "totalResults" field on the QueryJobsResponse. We don't know the answer to that on the dsub side though, so that would need to be an optional part of the API and we'd still need to maintain the existing behavior.

Reply:
ok, thanks for the info!